openeo-geotrellis/src/test/scala/org/openeo/geotrellis/layers/FileLayerProviderTest.scala

```diff
@@ -74,4 +78,105 @@ class FileLayerProviderTest {
 
     assertEquals(25, physicalMean)
   }
+
+  @Test
+  def sparsePartitionerTest(): Unit = {
```

Comment:
is it an option to split up this test into multiple smaller tests?

Reply:
I used one large test with a large dataset/extent to really test the difference between the two partitioners when there are many partitions. I was reluctant to split it up because then the large RDD would have to be loaded in for every test. 

However, looking at it again, I can definitely split the test up and use multiple smaller datasets.